Assets/_Scripts/Managers/GameManager.cs

```diff
@@ -85,15 +83,6 @@ protected override void Awake()
         EnemyManager.Instance.AllEnemiesKilled += OnAllEnemiesKilled;
     }
 
-    private void OnDisable()

```

Comment:
Hi There,
Would you mind explaining why we are removing this?
I am struggling to understand.

Thank you

Reply:
These were just comments I previously had that I let through into my previous pull request. I wanted to keep in mind that I'll need to figure out a way to clean up the EnemyManager appropriately when the scene ends but after today's discussions regarding how the EnemyManager is reset, this is actually no longer needed.